crates/nostr/src/nips/nip26.rs

```diff
@@ -9,32 +9,96 @@ use bitcoin_hashes::sha256::Hash as Sha256Hash;
 use bitcoin_hashes::Hash;
 use secp256k1::schnorr::Signature;
 use secp256k1::{KeyPair, Message, XOnlyPublicKey};
+use serde_json::{json, Value};
 
+#[cfg(feature = "base")]
+use crate::event::Event;
 use crate::key::{self, Keys};
 use crate::SECP256K1;
 
+use core::fmt;
+use std::str::FromStr;
```

Comment:
Can you move this on top of external crates?
In this library I use this format: import std modules, import external crates, import internal modules.